src/malloc_io.c

```diff
@@ -474,7 +471,6 @@ malloc_vsnprintf(char *str, size_t size, const char *format, va_list ap) {
 				assert(uwidth != UINTMAX_MAX || get_errno() !=
 				    ERANGE);
 				width = (int)uwidth;
-				first_width_digit = false;
```

Comment:
To confirm, this is the dead store we are getting rid of, right?

Reply:
Yes. The others were pretty straightforward, but this one took me a bit to convince myself it was correct. It _is_ a dead store, but unlike the others I'm not entirely sure that just removing it is the correct thing to do.